nevergrad/optimization/evolution_ops/rankers.py

```diff
@@ -0,0 +1,224 @@
+import numpy as np
```

Comment:
I guess this file can be moved and renamed as nevergrad/optimization/multiobjective/nsga2.py to clarify things, and same thing for the corresponding test file (the one that is currently name test_nsga2.py should be removed, it's actually not testing the feature, see comment below)